File: JS/CheckifStringsCanBeMadeEqualWithOperationsII.test.js
/**
 * @version JAVASCRIPT ECMAScript 6 
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ node NomeDaClasse.test
 */
import TEST from 'tape'
import { checkStrings } from './CheckifStringsCanBeMadeEqualWithOperationsII.js'

TEST('Starting CheckifStringsCanBeMadeEqualWithOperationsII test...', (t) => {
    //Test 1
    let want = true
    let got = checkStrings( "abcdba", "cabdab" )
    t.assert( want === got, "Expect: "+want)
    //Test 2
    want = false
    got = checkStrings( "abe", "bea" )
    t.assert( want === got, "Expect: "+want)

    t.end()
})

File: JAVA/CheckifStringsCanBeMadeEqualWithOperationsIITest.java
/** 
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ mvn clean test -Dtest=your.package.TestClassName
 */ 
package JAVA;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class CheckifStringsCanBeMadeEqualWithOperationsIITest {

    @Test
	public void testCheckifStringsCanBeMadeEqualWithOperationsII() throws Exception {
        //Test 1
        boolean want = true;
        boolean got = CheckifStringsCanBeMadeEqualWithOperationsII.checkStrings( "abcdba", "cabdab" );
        assertEquals(want, got);
        //Test 2
        want = false;
        got = CheckifStringsCanBeMadeEqualWithOperationsII.checkStrings( "abe", "bea" );
        assertEquals(want, got);
	}

}
